pyecharts/custom/page.py

```diff
@@ -66,7 +66,9 @@ def _repr_html_(self):
             components += chart._render_notebook_component_()
 
         require_config = conf.CURRENT_CONFIG.produce_require_configuration(
-            dependencies, self._jshost)
+            dependencies,
+            conf.CURRENT_CONFIG.get_current_jshost_for_jupyter(self._jshost)
```

Comment:
why not to replace 'conf.CURRENT_CONFIG.get_current_jshost_for_jupyter(self._jshost)' with 'self._jshost'?